README.md

```diff
@@ -3,11 +3,29 @@ Reports showing usage of LEV.
 
 ## Quick start
 ```
-npm i
+npm install --legacy-peer-deps
 npm test
 DB_USER=username DB_PASSWORD=password DB_HOST=hostname DB_PORT=port DB_DB=name npm start
 ```
 
+### Note
+The --legacy-peer-deps parameter is required by Node 16 as it has much stricter peer dependency checking than Node 12.
+
+## Local Developement & Testing
+### Test against an environment
+```
+POSTGRES_DB=xxx POSTGRES_USER=xxx POSTGRES_PASSWORD=xxx POSTGRES_SSL=true NODE_TLS_REJECT_UNAUTHORIZED=0 npm start
+```
+#### Note
+- Insert appropriate values for the environment variables.
+- Start the postgres proxy for the environment
+- Set NODE_TLS_REJECT_UNAUTHORIZED=0 to allow self signed certificates during local testing
+
+### Test against a locally running database
+```
+docker compose up --build
```

Comment:
docker-compose with the hyphen is required

Reply:
Depends which version of Decker Desktop you have, docker-compose command has now been included in docker command.

Reply:
You live and learn...